Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Set default values for build arguments
-ARG DEFRA_VERSION=1.2.10
-ARG BASE_VERSION=16.13.0-alpine3.14
+ARG DEFRA_VERSION=1.2.12
```

Comment:
this version change was missed last time, as it's a default it had no impact on the built image, just the version when built locally.